Add info about manual trigger of conda update to release guide (#859)
napari/docs: docs/developers/coredev/release.md

### New releases

> Once the PyPI release is available, the `conda-forge` bots will submit a PR to `conda-forge/napari-feedstock` within about twelves hours. Alternatively, to speed up the process, you may open an issue [following the version update instructions in this template](https://github.com/conda-forge/napari-feedstock/issues/new?template=2-bot-commands.yml) to manually trigger the update. The release manager may not have push/merge rights to the feedstock, so ask a recipe maintainer for assistance. Merging the bot PR to `main` will trigger the `conda-forge` release.

Reviewer: Moved template instructions to here
```suggestion
Once the PyPI release is available, the `conda-forge` bots will submit a PR to `conda-forge/napari-feedstock` within about twelves hours. Alternatively, to speed up the process, you may open an issue with title [`@conda-forge-admin, please update version`](https://github.com/conda-forge/napari-feedstock/issues/new?template=2-bot-commands.yml) to manually trigger the bot. The release manager may not have push/merge rights to the feedstock, so ask a recipe maintainer for assistance. Merging the bot PR to `main` will trigger the `conda-forge` release.
```
